lib/aws_command.rb

```diff
@@ -7,7 +7,7 @@ module OpenShift
   module Ops
     class AwsCommand < Thor
       # WARNING: we do not currently support environments with hyphens in the name
-      SUPPORTED_ENVS = %w(prod stg int tint kint test jhonce amint tdint lint)
+      SUPPORTED_ENVS = %w(prod stg int tint kint test jhonce amint tdint lint jdetiber)
```

Comment:
I don't see an issue with it, though I think I would much rather we add a flag for both cluster.sh and cloud.rb to allow for an arbitrary string as the environment (something like --dev).

For anything that would be used commonly I would much rather see command line options/flags instead of having to use env variables. Keeping documentation up to date for env variables and describing the sometimes complex relationships between different env variables is much more work than having it done through the option parser and the validation the script already has to do.